db/event_manager_test.go

```diff
@@ -104,7 +103,7 @@ func TestDocumentChangeEvent(t *testing.T) {
 	//Raise events
 	for i := 0; i < 10; i++ {
 		body, docid, channels := eventForTest(i)
-		bodyBytes, _ := base.JSONMarshal(body)
+		bodyBytes, _ := base.JSONMarshalCanonical(body)
```

Comment:
Is it necessary to change all JSONMarshal usages in event_maanger_test?  It doesn't look like all tests are trying to assert the event body.

Reply:
Yeah, you are right it's not required in all tests. The change has been reverted and corrected in the appropriate place.